src/parsing.jl

```diff
@@ -118,7 +118,10 @@ if VERSION >= v"0.7.0-DEV.5125"
         iterate(pqv, (sv, firstindex(sv)))
```

Comment:
Sorry, missed this last time, but you need to actually iterate `sv` in this function; `firstindex` is not guaranteed to be the iteration state.